src/Tizen.NUI.WindowSystem/src/public/QuickPanelClient.cs

```diff
@@ -118,6 +119,39 @@ public class QuickPanelClient : IDisposable
             }
         }
 
+        private event EventHandler<int> _rotationChanged;
+
+        /// <summary>
+        /// Emits the event when the rotation(orientation) of the quickpanel service window is changed.
+        /// The value of the event argument represents the rotation angle in degrees.
+        /// </summary>
+        /// <exception cref="ArgumentException">Thrown when failed of invalid argument.</exception>
+        /// <since_tizen> 12 </since_tizen>
+        public event EventHandler<int> RotationChanged
+        {
+            add
+            {
+                if(_rotationChanged == null)
+                {
+                    OrientationChanged += OnRotationChanged;
+                }
+                _rotationChanged += value;
+            }
+            remove
+            {
+                _rotationChanged -= value;
+                if(_rotationChanged == null)
+                {
+                    OrientationChanged -= OnRotationChanged;
+                }
+            }
+        }
+
+        private void OnRotationChanged(object sender, Window.WindowOrientation e)
```

Comment:
I would expect that current instance will always be sender and there is no need for sender parameter, but it's not related to API

Reply:
yes, you are right. I would not change because it is private method.